docs/examples/external-dns.yaml

```diff
@@ -50,6 +50,8 @@ spec:
         app: external-dns
     spec:
       serviceAccountName: external-dns
+      securityContext:
+        fsGroup: 1337
```

Comment:
The referenced external-dns issue PR recommends [`fsGroup: 65534`](https://github.com/kubernetes-sigs/external-dns/pull/1185/files#diff-89e5478cb30ea6ae2a798bde1f237a693ddeee6437f42db8be87b600b2032f4dR163) which would also make this example yaml match with the example yaml in their [AWS documentation](https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/aws.md#manifest-for-clusters-with-rbac-enabled).

I don't think if it matters, but in case people want to use `istio` [their requirements](https://istio.io/latest/docs/ops/deployment/requirements/) specifically mention that "_Application UIDs: Ensure your pods do not run applications as a user with the user ID (UID) value of 1337._"

Reply:
The upstream external-dns from the kubernetes sig also recommends 65534 https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/aws.md#manifest-for-clusters-with-rbac-enabled